crossgrade/README.md

```diff
@@ -1,4 +1,4 @@
-# How to run the upgrade test
+# How to run the crossgrade test
```

Comment:
First time I've heard the term crossgrade. Can we please define it?

Reply:
yeah, I'm not sure which word to use here. Basically I want to include the meaning for both upgrade and downgrade. Maybe just update it to upgrade and downgrade?

Reply:
Offline discussed with @brian-avery, we think `upgrade_downgrade` is clearer than defining a new term in the README here.